### PR TITLE
Fix the crash occuring in VSC due to lack of PreferencesManager

### PR DESCRIPTION
**Pull request by hsiwkin**
[Issue] N/A
[Problem] App were crashing every time PreferencesManager is used.
[Solution] I've created methods that were lacking in PreferencesManager

Signed-off-by: Hubert Siwkin <h.siwkin@samsung.com>

### CODE REVIEW

File: design-editor/src/system/stage-manager.js [hunk 1]
```diff
@@ -236,10 +236,10 @@ class StageManager {
      * @param toggleEditor
```
Comment (korneliakobiela) [inline design-editor/src/system/stage-manager.js]: Please, correct documentation according to the current function state

File: design-editor/src/utils/utils.js [hunk 2]
```diff
@@ -69,10 +69,12 @@ module.exports = {
 	 * @param {object} bracketsEditor
```
Comment (korneliakobiela) [inline design-editor/src/utils/utils.js]: Please, correct documentation according to the current function state

File: design-editor/src/utils/utils.js [hunk 3]
```diff
@@ -69,10 +69,12 @@ module.exports = {
...
 	 * @returns {string} boolean
 	 */
-	isDemoVersion(bracketsEditor) {
+	isDemoVersion() {
+		const bracketsEditor  = this.checkGlobalContext('brackets');
 		if (bracketsEditor) {
```
Comment (korneliakobiela) [inline design-editor/src/utils/utils.js]: I think we don't need this 'if' here. In VSC we have mocked brackets API and assign it to 'window.brackets'

File: design-editor/src/utils/utils.js (hunk 2 above)
Comment (hsiwkin) [reply design-editor/src/utils/utils.js]: fixed

File: design-editor/src/utils/utils.js (hunk 3 above)
Comment (hsiwkin) [reply design-editor/src/utils/utils.js]: right, I've removed it

**Pull request merged**